templates/repo/branch/list.tmpl

```diff
@@ -7,6 +7,11 @@
 		{{if .DefaultBranchBranch}}
 			<h4 class="ui top attached header gt-mt-4">
 				{{.locale.Tr "repo.default_branch"}}
+				{{if and $.IsWriter (not $.Repository.IsArchived) (not .IsDeleted)}}
+				<a role="button" class="right" href="{{.RepoLink}}/settings/branches" data-tooltip-content="{{.locale.Tr "repo.settings.branches.switch_default_branch"}}">
+					{{svg "octicon-arrow-switch" 16 "gt-h-100"}}
```

Comment:
If this is for vertical alignment, do instead `gt-df gt-ac` on the parent node.

Reply:
Added in https://github.com/go-gitea/gitea/pull/24379/commits/eadbaf467a51eeae80c3dff59186dffdc5607d62, also added `gt-sb` for justify-content